village/src/components/SmurfForm.js

```diff
@@ -12,42 +13,55 @@ class SmurfForm extends Component {
 
   addSmurf = event => {
     event.preventDefault();
-    // add code to create the smurf using the api
+    const smurf = { ...this.state }
+    const url = 'http://localhost:3333/smurfs';
+    
 
+    axios.post(url, smurf)
+      .then(res => {
+        this.setState({
+          smurf: res.data
+        })
+        this.props.history.push("/");
+        window.location.reload();
```

Comment:
I would use setState to update the page instead of reloading because window.location.reload causes the whole page to refresh again.